extensions/adobe/experience/bizible/bizible-account-details.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/bizible-account-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Bizible Account Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/account"],
+  "meta:extends": [
```

Comment:
Not sure we should do this. Instead this mixin can just introduce two new fields and be included with the other two mixins. The goal is not to create a mixin that replaces the others.